pkg/kademlia/kademlia_planet_test.go

```diff
@@ -39,3 +47,52 @@ func TestRequestInfo(t *testing.T) {
 		require.Equal(t, node.Local().Restrictions.GetFreeBandwidth(), info.GetCapacity().GetFreeBandwidth())
 	})
 }
+
+func TestPingTimeout(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 0,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+
+		{
```

Comment:
Unnecessary code block.

Reply:
also @egonelbre - is this test redundant with the dialer tests I added? I wanted to cover the endpoint functions (like Query) but realized that no errors are returned so it's kind of hard to check that the context deadline was hit. So I went down a level, but the `kademlia.Ping` function basically just wraps `dialer.PingNode` which already has a timeout test.